fdbserver/MoveKeys.actor.cpp

```diff
@@ -28,6 +28,40 @@
 using std::min;
 using std::max;
 
+// in-memory flag to disable DD
+bool ddEnabled = true;
+UID ddEnabledStatusUID = UID();
```

Comment:
I was resistant to the initial piece of global state that you added for snapshotting, and it seems to have only increased since.  In this particular case, it's possible for multiple data distributors to be alive at once, and I'd be concerned about what might happen with this.  Overall, I'd be happy to see the global state get refactored away post 6.2